### PR TITLE
feature/account deletion

### PR DESCRIPTION
**Pull request by aarontorres0**

### CODE REVIEW

File: assets/icons8-settings-96.png
Comment (jlipscomb071) [inline assets/icons8-settings-96.png:1]: Why was this removed?

Comment (aarontorres0) [reply assets/icons8-settings-96.png:1]: No longer using this for the settings icon. Instead I import a cog icon from react-native-vector-icons/FontAwesome which should be more scalable on different devices

**Review by oosinloye — APPROVED** [review]
lgtm
